app/src/main/java/com/napzak/market/presentation/marketinfo/MarketInfoViewModel.kt

```diff
@@ -122,21 +122,17 @@ class MarketInfoViewModel @Inject constructor(
     fun getMarketInformation() = viewModelScope.launch {
         marketInfoRepository.fetchMarketInfo(uiState.value.storeId)
             .onSuccess { marketInfo ->
-                if (marketInfo.storeId==null) {
-                    updateLoadMarketInfoState(UiState.Empty)
-                } else {
-                    updateLoadMarketInfoState(
-                        UiState.Success(
-                            MarketUiInformation(
-                                storeNickname = marketInfo.storeNickname,
-                                storeDescription = marketInfo.storeDescription,
-                                storePhoto = marketInfo.storePhoto,
-                                storeCover = marketInfo.storeCover,
-                                genrePreferenceList = marketInfo.genrePreferenceList,
-                            )
+                updateLoadMarketInfoState(
+                    UiState.Success(
+                        MarketUiInformation(
+                            storeNickname = marketInfo.storeNickname,
+                            storeDescription = marketInfo.storeDescription,
+                            storePhoto = marketInfo.storePhoto,
+                            storeCover = marketInfo.storeCover,
+                            genrePreferenceList = marketInfo.genrePreferenceList,
                         )
```

Comment:
p2: 이렇게 바꿔보시는건 어떤가요?
```suggestion
            UiState.Success(
                  with(marketInfo) {
                        storeNickname = storeNickname,
                        storeDescription = storeDescription,
                        storePhoto = storePhoto,
                        storeCover = storeCover,
                        genrePreference = genrePreferenceList,
                  }
            )
```